app/aws-lsp-codewhisperer-runtimes/scripts/package.sh

```diff
@@ -105,7 +106,7 @@ done
 cd ./build/private/bundle
 for config in "${configs[@]}"; do
     cd ${config}
-    zip -r ../../../../$ARCHIVES_DIR/${config}/win-x64/servers.zip .
+    zip -r ../../../../$ARCHIVES_DIR/${config}/win-x64/servers.zip . -x 'registry.node' '*/registry.node'
```

Comment:
We shouldn't need this because your change on line 74 already adds the file to the zip. If we add more entries to the `configs` array (not likely) we can bundle if or as needed.

Reply:
Removing this line causes the `registry.node` I added to be overwritten by a different copy (I believe it's the one emitted by node-loader under `build/private/bundle/**`)

Reply:
Oh I see, `npm run webpack:prod` places the file into `app\aws-lsp-codewhisperer-runtimes\build\private\bundle\agent-standalone\registry.node`

So what we could do is update `ci:generate:agent-standalone`:
- add a new script into app\aws-lsp-codewhisperer-runtimes\package.json called `copy:native-deps:agent-standalone`
- `copy:native-deps:agent-standalone` will be similar to `copy:resources:agent-standalone`
- `copy:native-deps:agent-standalone` will copy registry.node from _bundle-assets into `app\aws-lsp-codewhisperer-runtimes\build\private\bundle\agent-standalone\registry.node`
- update `ci:generate:agent-standalone` to call `npm run copy:native-deps:agent-standalone` before the call is made to `npm run copy:resources:agent-standalone`

Reply:
This makes sense, but won't this also use this `registry.node` file in the `servers.zip` for other platforms instead of the one generated from webpacking? Do we want to do that?